pyaerocom/scripts/cams2_82/cli.py

```diff
@@ -0,0 +1,239 @@
+from __future__ import annotations
+
+import logging
+import multiprocessing as mp
+from copy import deepcopy
+from collections.abc import Iterator
+from datetime import date, datetime, timedelta
+from pathlib import Path
+from typing import Optional
+
+import typer
+
+from pyaerocom import change_verbosity, const
```

Comment:
const is deprecated
```
Use of pyaerocom.const is deprecated. Please use Config.get_instance() or pyaerocom.config instead.
```
(I know, we are still using it everywhere else, but we shouldn't add it in new places.)